push-the-button.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash -ex
+
+k8s_provider="${KUBEVIRT_PROVIDER:-k8s-1.18}"
+kubevirt_src_code="${KUBEVIRT_CODE:-$HOME/kubernetes/kubevirt/}"
```

Comment:
we should use standard location of kubevirtci path as default:
`kubevirt_src_code="${KUBEVIRT_CODE:-$GOPATH/src/github.com/kubevirt/kubevirtci}"`

Reply:
Done.